core/src/main/scala/org/apache/spark/status/protobuf/KVStoreProtobufSerializer.scala

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.status.protobuf
 
-import org.apache.spark.status.{ApplicationEnvironmentInfoWrapper, ApplicationInfoWrapper, JobDataWrapper, TaskDataWrapper}
+import org.apache.spark.status.{ApplicationEnvironmentInfoWrapper, ApplicationInfoWrapper, JobDataWrapper, RDDStorageInfoWrapper, TaskDataWrapper}
```

Comment:
Let's change this to `org.apache.spark.status._` to save future troubles.

Reply:
done